MoreLinq/AggregateRight.cs

```diff
@@ -47,10 +47,7 @@ public static TSource AggregateRight<TSource>(this IEnumerable<TSource> source,
             if (source == null) throw new ArgumentNullException(nameof(source));
             if (func == null) throw new ArgumentNullException(nameof(func));
 
-            var list
```

Comment:
Why not also in the other overload?

https://github.com/morelinq/MoreLINQ/blob/5048831832e3928fa1682e510fb3f330e2710f47/MoreLinq/AggregateRight.cs#L89-L91